src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style.length > 0);
+
+  // Converte cada linha em uma entrada de objeto
+  const stylesObject = {};
+
+  stylesArray.forEach((style) => {
```

Comment:
[CODE PRACTICE] - Although using `forEach` is not incorrect, consider using `reduce` to build the styles object directly from the array of styles. This would align with the functional programming practices recommended in the checklist.